services/feed-manager-service/feed-manager-controller/src/main/java/com/thinkbiganalytics/feedmgr/rest/controller/FeedRestController.java

```diff
@@ -829,5 +850,18 @@ public Response uploadFile(@PathParam("feedId") String feedId,
         }
         return Response.ok("").build();
     }
+
+    private PageRequest pageRequest(Integer start, Integer limit, String sort) {
+        if (StringUtils.isNotBlank(sort)) {
+            Sort.Direction dir = Sort.Direction.ASC;
+            if (sort.startsWith("-")) {
+                dir = Sort.Direction.DESC;
+                sort = sort.substring(1);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "sort". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1226)